EvilCastle/EvilCastle/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+
+namespace TextAdventure
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Welcome to the castle of the evil King Alexander!");
+            Console.WriteLine();
+            Console.WriteLine("Your quest is to defeat the king before he conquers the land with his undead army.");
+            /*Console.WriteLine("Pick a number: 1, 2 or 3");
+            string input = Console.ReadLine();
+            if (input == "1")
+            {
+                Console.WriteLine("correct");
+            }
+            */
+            Console.WriteLine("Press enter if you accept this quest, otherwise leave and die like the coward you are, you freak!");
+            Console.ReadLine();
+
+            Console.WriteLine("You enter the castle, and you find yourself staring at a undead soldier who is running towards you!");
+            Console.WriteLine("Press 1 to stab him in the head");
+            Console.WriteLine("Press 2 to dodge and slash him in the back");
+            Console.WriteLine("Press 3 to run away and go home");
+            string input = Console.ReadLine();
+            if (input == "1")
```

Comment:
I see multiple nested if statements, i hope you researched it to truly understand the capability of them all